src/lm/tools/copilotRemoteAgentTool.ts

```diff
@@ -127,7 +127,10 @@ export class CopilotRemoteAgentTool implements vscode.LanguageModelTool<CopilotR
 			};
 			const buffer: Buffer = Buffer.from(JSON.stringify(preferredRendering));
 			const data: Uint8Array = Uint8Array.from(buffer);
-			lmResult.push(new vscode.LanguageModelDataPart2(data, 'application/pull-request+json', [vscode.LanguageModelPartAudience.User]));
+
+			// API might not be available for tests, this guarantees we are still able to test
+			const userAudience = (vscode as any).LanguageModelPartAudience?.User ?? 1;
```

Comment:
@alexr00 Had to do this because the value was not available during runtime for the tests, not sure if there's a better way to make sure the value is available